docs/equinix/index.md

```diff
@@ -1,2 +1,171 @@
-# Equinix Metal Cookbooks
-### Recipes for using and deploying DLTs using Sextant on Equinix Metal
\ No newline at end of file
+# Equinix Metal Cookbook
+
+Recipe for using and deploying blockchain networks using Sextant on Equinix Metal
```

Comment:
Technically this is laying the groundwork as here you are really providing instructions for (a) creating Kubernetes clusters and (b) installing Sextant (currently missing from this - see comment below)

Reply:
Changed.